app/default/content_routes.py

```diff
@@ -54,3 +57,27 @@ def contact_us():
 def cookie_policy():
     current_app.logger.info("Cookie policy page loaded.")
     return render_template("cookie_policy.html")
+
+
+@content_bp.route("/privacy", methods=["GET"])
+def privacy():
+    current_app.logger.info("Privacy_notice page loaded.")
+    round_data = get_round_data_fail_gracefully(
+        Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID
+    )
+
+    privacy_notice_url = (
+        round_data.privacy_notice
+        if hasattr(round_data, "privacy_notice")
+        else None
+    )
```

Comment:
This may be cleaner:

```suggestion
    privacy_notice_url = getattr(round_data, "privacy_notice", None)
```